### DISCUSSION TITLE
Disadvantages of placing object data in page YAML?

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#922] [Get Help]
What are the advantages/disadvantages of adding object data directly to the page YAML, rather than keeping it all in the objects.yaml file? I would love to hear from people who have tried both. I'm about to begin typesetting a collection catalogue that will eventually have short entries on nearly 2,000 Indonesian textiles. I'm thinking about placing the object data in the page YAML for each entry so I can see the title and accession number right on each entry page while working in Atom. 

The other collection catalogue I generated has all of the object data in the objects.yaml file. 

Here is the info in the v1 documentation on these two options:
https://quire.getty.edu/docs-v1/collection-catalogues/#create-object-pages

Thank you!

**Comment by Erin-Cecele** [comment]
For the most part, it's just a matter of preference. That said, keeping the object entry information centralized in the `objects.yaml` file means you can do more programmatic things like exporting to CSV or using Liquid logic.

**Reply by zsofiaj** [reply]
Thank you! What are the contexts in which one might want to export to CSV or use Liquid logic with data from the objects.yaml file?

**Reply by Erin-Cecele** [reply]
Sometimes people like to export information out of their CMS into a CSV file that can then be converted to YAML. See my recent newsletter for tips: https://newsletters.getty.edu/campaigns/reports/viewCampaign.aspx?d=t&c=D464D9637E65C4DF&ID=C622C3D425D042482540EF23F30FEDED&temp=False&tx=0&source=Report.

I actually used this recently for a project. I needed the data in CSV so I could do a complex V-LookUp, and it worked like a charm.

I'm less well-versed in Liquid, but from what I've seen, it's pretty magical. We did use it recently to create a Glossary from a glossary.yaml file, that was also being used for pop-up definitions.
